docs/modules/ROOT/pages/api/multitoken.adoc

```diff
@@ -0,0 +1,244 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:multiToken-guide: xref:multiToken.adoc[MultiToken guide]
+:erc1155-metadata: xref:https://eips.ethereum.org/EIPS/eip-1155#metadata[ERC1155Metadata]
+
+= MultiToken
+
+This module provides the full MultiToken module API.
+
+TIP: For an overview of the module, read the {multiToken-guide}.
+
+== Core
+
+[.hljs-theme-dark]
+```ts
+import "openzeppelin-compact/multi-token" prefix MultiToken_;
+```
+
+[.contract]
+[[MultiToken]]
+=== `++MultiToken++` link:https://github.com/OpenZeppelin/compact-contracts/blob/main/contracts/multiToken/src/MultiToken.compact[{github-icon},role=heading-link]
+
+[.contract-index]
+.Circuits
+--
+
+[.sub-index#MultiTokenModule]
+* xref:#MultiTokenModule-initialize[`++initialize(uri_)++`]
+* xref:#MultiTokenModule-uri[`++uri(id)++`]
+* xref:#MultiTokenModule-balanceOf[`++balanceOf(account, id)++`]
+* xref:#MultiTokenModule-setApprovalForAll[`++setApprovalForAll(operator, approved)++`]
+* xref:#MultiTokenModule-isApprovedForAll[`++isApprovedForAll(account, operator)++`]
+* xref:#MultiTokenModule-transferFrom[`++transferFrom(from, to, id, value)++`]
+* xref:#MultiTokenModule-_transfer[`++_transfer(from, to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeTransferFrom[`++_unsafeTransferFrom(from, to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeTransfer[`++_unsafeTransfer(from, to, id, value)++`]
+* xref:#MultiTokenModule-_setURI[`++_setURI(newURI)++`]
+* xref:#MultiTokenModule-_mint[`++_mint(to, id, value)++`]
+* xref:#MultiTokenModule-_unsafeMint[`++_unsafeMint(to, id, value)++`]
+* xref:#MultiTokenModule-_burn[`++_burn(from, id, value)++`]
+* xref:#MultiTokenModule-_setApprovalForAll[`++_setApprovalForAll(owner, operator, approved)++`]
+--
```

Comment:
I think we should include the `_update` circuit and refactor later to align with #179 / the proposed design template

Reply:
Done!